assets/src/scss/layout/_featured-action.scss

```diff
@@ -0,0 +1,94 @@
+.featured-action-section {
+  margin-top: 50px;
```

Comment:
Could we use spacing CSS variables? Same to all `padding`/`margin`/`gap` properties!

Reply:
Yes, still updating the code, thanks for the reminder